src/style.css

```diff
@@ -12,15 +12,15 @@
 }
 
 .stars--1 .stars__star:first-child,
-.stars--2 .stars__star:nth-last-child(5),
-.stars--2 .stars__star:nth-last-child(4),
-.stars--3 .stars__star:nth-last-child(5),
-.stars--3 .stars__star:nth-last-child(4),
-.stars--3 .stars__star:nth-last-child(3),
-.stars--4 .stars__star:nth-last-child(5),
-.stars--4 .stars__star:nth-last-child(4),
-.stars--4 .stars__star:nth-last-child(3),
-.stars--4 .stars__star:nth-last-child(2),
-.stars--5 .stars__star:nth-last-child(n) {
+.stars--2 .stars__star:nth-child(1),
+.stars--2 .stars__star:nth-child(2),
+.stars--3 .stars__star:nth-child(1),
+.stars--3 .stars__star:nth-child(2),
+.stars--3 .stars__star:nth-child(3),
+.stars--4 .stars__star:nth-child(1),
+.stars--4 .stars__star:nth-child(2),
+.stars--4 .stars__star:nth-child(3),
+.stars--4 .stars__star:nth-child(4),
+.stars--5 .stars__star:nth-child(n) {
```

Comment:
you can leave only five selectors if use `nth-child` with `-n`, like this
```
.stars--1 .stars__star:nth-child(-n+1),
.stars--2 .stars__star:nth-child(-n+2),
...
```

Reply:
I've already done)